hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -548,6 +558,10 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
                       <Button>Display Limit: {displayLimit}</Button>
                     </Dropdown>
                   </div>
+                  &nbsp;&nbsp;&nbsp;&nbsp;
+                  <Tooltip placement="rightTop" title="In Below chart we have achieved visibility for all small entities using additional block size.">
```

Comment:
Can we change this message to something like:
```
Additional block size added for viewing small entity sizes. Please refer to pie-chart tooltip for exact size information
```
@ArafatKhan2198 @dombizita maybe you guys could suggest a better statement?

Reply:
Done changes suggested by you.